### DISCUSSION TITLE
Raffreddamento rpi4

### DISCUSSION THREAD

**Discussion by lucaxxaa** [indomus/forum#439] [GENERALE - Discussioni generiche]
Ciao a tutti . Ho ordinato questo dissipatore passivo per rpi4 che sembra uno dei migliori

https://www.sensoreq.com/en/coolipi-heatsink/29-coolipi-4b-chladic-cern.html

Volevo eventualmente abbinare una ventola noctuna.
È possibile usare ventola che ha tre pin con controllo velocità ( PWM) e gestirla da HA, per farla partire solo sopra una certa soglia?

https://amzn.to/3gR9j2D

La ventola la monterei sora il dissipatore

**Comment by nromagna** [comment]
Ciao io ho preso proprio quella ventola li con un PI4. So passato, oltre al rumore assordante della ventola originale ad un "soffio" appena percettibile, da 42° a 37°. Ho dovuto modificare il case, ma con un po' di manualità (e credi io NON ne ho!) è fattibilissimo.

**Reply by lucaxxaa** [reply]
Ah ok, ma è possibile controllarla con home assistant cioè farla partire solo dopo una certa temperatura? E poi i tre PIN si collegano al Raspberry

Che uso già un dissipatore passivo e non voglio che la ventola sia sempre attiva
